src/MetaboxRegister.php

```diff
@@ -35,120 +34,11 @@ private function hooks() {
 	 * Register P4 meta box.
 	 */
 	public function register_p4_meta_box() {
-		$this->register_meta_box_header();
 		$this->register_meta_box_post();
 		$this->register_meta_box_open_graph();
 		$this->register_meta_box_campaign();
 	}
 
-	/**
-	 * Register Page Header meta box.
-	 */
-	public function register_meta_box_header() {
-
-		$p4_header = new_cmb2_box(
-			[
-				'id'           => 'p4_metabox',
-				'title'        => __( 'Page Header Fields', 'planet4-master-theme-backend' ),
-				'object_types' => [ 'page', 'campaign' ], // Post type.
-				'show_in_rest' => WP_REST_Server::READABLE,
-			]
-		);
-
-		$p4_header->add_field(
-			[
-				'name'         => __( 'Header Title', 'planet4-master-theme-backend' ),
-				'desc'         => __( 'Header title comes here', 'planet4-master-theme-backend' ),
-				'id'           => 'p4_title',
-				'type'         => 'text_medium',
-				'show_in_rest' => WP_REST_Server::READABLE,
-			]
-		);
-
-		$p4_header->add_field(
-			[
-				'name'         => __( 'Header Subtitle', 'planet4-master-theme-backend' ),
-				'desc'         => __( 'Header subtitle comes here', 'planet4-master-theme-backend' ),
-				'id'           => 'p4_subtitle',
-				'type'         => 'text_medium',
-				'show_in_rest' => WP_REST_Server::READABLE,
-			]
-		);
-
-		$p4_header->add_field(
-			[
-				'name'         => __( 'Header Description', 'planet4-master-theme-backend' ),
-				'desc'         => __( 'Header description comes here', 'planet4-master-theme-backend' ),
-				'id'           => 'p4_description',
-				'type'         => 'wysiwyg',
-				'options'      => [
-					'textarea_rows' => 5,
-					'media_buttons' => false,
-				],
-				'show_in_rest' => WP_REST_Server::READABLE,
-			]
-		);
-
-		$p4_header->add_field(
-			[
-				'name'       => __( 'Header Button Title', 'planet4-master-theme-backend' ),
-				'desc'       => __( 'Header button title comes here', 'planet4-master-theme-backend' ),
-				'id'         => 'p4_button_title',
-				'type'       => 'text_medium',
-				'show_on_cb' => [ $this, 'is_not_campaign_post' ],
```

Comment:
As we are removing the these metabox fields, we need to remove [this](https://github.com/greenpeace/planet4-master-theme/blob/master/src/MetaboxRegister.php#L337) related function as well.

Reply:
Good catch! Removed 👍